src/main/java/com/google/idea/perf/TracerView.kt

```diff
@@ -72,14 +74,17 @@ class TracerView(parentDisposable: Disposable) : JBPanel<TracerView>() {
     private val controller: TracerController = TracerController(this, parentDisposable)
     val progressBar: JProgressBar
     val listView = TracepointTable(TracepointTableModel())
+    val refreshTimeLabel: JBLabel
 
     init {
         preferredSize = Dimension(500, 500) // Only applies to first open.
         layout = BoxLayout(this, BoxLayout.Y_AXIS)
 
+        val maximumChildSize = Dimension(Integer.MAX_VALUE, minimumSize.height)
```

Comment:
In Kotlin, when you have code like
```kotlin
JBTextField().apply {
    // ...
}
```
then anything inside the `apply` block will have the `JBTextField` as a receiver. See https://kotlinlang.org/docs/reference/scope-functions.html#apply

So, before this commit, `minimumSize` referred to `commandLine.minimumSize`, whereas here it refers to the minimum size of the `TracerView`.

(Some would argue this is a good example of why to avoid `apply()` altogether, but alas it is fun to use.)

In this case, I think duplicating `(Integer.MAX_VALUE, minimumSize.height)` is OK.